.github/scripts/delete-aws-resources.sh

```diff
@@ -41,24 +41,21 @@ function delete_old_aws_resources() {
   # Empty the file
   truncate -s 0 $DELETED_RESOURCES_FILE
 
-  for resource_type in ${RESOURCE_TYPES//,/ }
-  do
-    aws cloudcontrol list-resources --type-name "$resource_type" --query "ResourceDescriptions[].Identifier" --output text | tr '\t' '\n' | while read identifier
-    do
-      aws cloudcontrol get-resource --type-name "$resource_type" --identifier "$identifier" --query "ResourceDescription.Properties" --output text | while read resource
-      do
-        resource_tags=$(jq -c -r .Tags <<< "$resource")
-        for tag in $(jq -c -r '.[]' <<< "$resource_tags")
-        do
-          key=$(jq -r '.Key' <<< "$tag")
-          value=$(jq -r '.Value' <<< "$tag")
-          if [[ "$key" == "$LABEL" && $((CURRENT_TIME - value)) -gt $MAX_AGE]]
-          then
-            echo "Deleting resource of type: $resource_type with identifier: $identifier"
-            echo "$identifier\n" >> $DELETED_RESOURCES_FILE
-            aws cloudcontrol delete-resource --type-name "$resource_type" --identifier "$identifier"
-          fi
-        done
+  for resource_type in ${RESOURCE_TYPES//,/ }; do
+    aws cloudcontrol list-resources --type-name "$resource_type" --query "ResourceDescriptions[].Identifier" --output text | tr '\t' '\n' | while read identifier; do
+      aws cloudcontrol get-resource --type-name "$resource_type" --identifier "$identifier" --query "ResourceDescription.Properties" --output text | while read resource; do
+        resource_tags=$(jq -c -r .Tags <<<"$resource")
+        if [[ "$resource_tags" != "null" && "$resource_tags" != "" ]]; then
```

Comment:
Added this because of this error: `jq: error (at <stdin>:1): Cannot iterate over null (null)`.